test/common/alerts_exist.go

```diff
@@ -232,6 +232,8 @@ var expectedRules = []alertsTestRule{
 			"RouterMeshConnectivityHealth",
 			"RouterMeshUndeliveredHealth",
 			"BrokerMemory",
+			"AddressCanaryCheckHealth",
+			"AddressCanaryHealth",
```

Comment:
@david-martin AMQ online 1.5.x is introducing to new alerts, is there anything we need to be doing/considering regarding this? Any additional manual tests we should be doing?

Reply:
If the alerts are 'critical' severity, they need to meet the CSSRE critical alert criteria.
https://issues.redhat.com/browse/CSSRE-1104
That is the only hard requirement I am aware of.

I'm following up on getting the criteria into a more accessible location. I think the failing test criteria & output can be improved so it only fails for new 'critical' severity alerts and links to the criteria. 
However, having this kind of test earlier in the CI process for product operators seems like a better place for this ultimately as it can be rectified much closer to development time.